### ISSUE TITLE
Accessibility issue with the new sidebar

### ISSUE THREAD

**Issue by javigomez** [joomla/joomla-cms#6063]
This is not an issue that we need to tackle right now, and we can probably work on it when we work for the release of Joomla 3.5 with the JLayout overrides. But I wanted to rise awareness that we are failing with the Contrast Accessibility requirements here:

![screen shot 2015-02-12 at 10 50 54](https://cloud.githubusercontent.com/assets/1375475/6165062/16db9f6c-b2a5-11e4-87bc-c592ee07c3f4.png)

cc: @phproberto

**Comment by infograf768** [comment]
Agree

**Comment by infograf768** [comment]
I guess giving it the same coulour as the words in the filters would fit.
I.e. #444 instead of #999

```
.j-toggle-sidebar-header {
    border-bottom: 1px solid #eee;
    border-top: 1px solid #d3d3d3;
    color: #444;
    margin: -11px 0 7px;
    padding: 10px 0 10px 19px;
}
```

**Comment by tottello** [comment]
Color #444 makes word 'Sidebar' too much like links below. Maybe bigger/bold (or without this word at all).

**Comment by JoomliC** [comment]
@javigomez I had used the same color HEX as for the footer status as an not-link label. (and issue with contrast in status bar too!).

But, i agree on working on standard colors for future admin template, which would be required as standard rules to follow (as it is existing now for code standards, and language strings) ;-)<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6063">issues.joomla.org/joomla-cms/6063</a>.</sub>

**Comment by nonumber** [comment]
See: https://github.com/joomla/joomla-cms/pull/6340#discussion_r25995964

**Comment by brianteeman** [comment]
Do we even need this text? Having it is a bit like adding a label to the menu to say it is a menu. Does it serve a purpose?

**Comment by nonumber** [comment]
That's what I said in the comment: https://github.com/joomla/joomla-cms/pull/6340#discussion_r25995964

**Comment by nonumber** [comment]
If it's ok, I can add the removal of the title in the no-gradient PR.
But that PR is getting larger and larger...

**Comment by JoomliC** [comment]
@nonumber yes, but with removal of title, your PR will be ready to commit, and  with this PR ready to be closed too! ;-)
So :+1:

**Comment by nonumber** [comment]
I think this can get closed in favour of https://github.com/joomla/joomla-cms/pull/6340
(if that gets in ;) )

**Comment by brianteeman** [comment]
Closing as #6340 was merged<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/6063">issues.joomla.org/joomla-cms/6063</a>.</sub>
